pkg/loader/loader.go

```diff
@@ -131,6 +131,36 @@ func (kinds *TroubleshootKinds) ToYaml() (string, error) {
 	return strings.Join(rawList, "---\n"), nil
 }
 
+// GetURIs dynamically extracts all the URIs from the troubleshoot kinds
+// .Spec.Uri field
+func (kinds *TroubleshootKinds) GetURIs() map[string]bool {
```

Comment:
I would keep this function internal to troubleshoot. Its an implementation utility function to assist updating specs, which callers at won't need to do